### DISCUSSION TITLE
REST API

### DISCUSSION THREAD

**Discussion by junaid33** [blankly-finance/blankly#84] [General]
I couldn't find documentation about how to use the REST API

**Comment by EmersonDove** [comment]
Hi,

The blankly REST calls are generally routed through any `.interface` call.

For example:
```python
import blankly

cb = blankly.CoinbasePro()

cb.market_order('BTC-USD', 'buy', .01)
```


You can also access the REST API our interface is built on with
```python
coinbase_pro.get_direct_calls().create_report()
```

The documentation should be available [here](https://docs.blankly.finance/core/exchange_interface). If you have suggestions for better names or way of pointing people towards the API we are open for suggestions!
